testsuite/src/Test/Tasty/Clash.hs

```diff
@@ -281,7 +281,7 @@ sequenceTests path (unzip -> (testNames, testTrees)) =
     where
       -- Make pattern for a single test
       pat :: TestName -> String
-      pat nm = "$0 == \"" ++ intercalate "." (reverse (nm:path)) ++ "\""
+      pat nm = "$0 ~ /" ++ intercalate "." (reverse (nm:path)) ++ "/"
```

Comment:
`sequenceTests` is used in multiple places, does this mean all types of test are now run sequentially?

Reply:
No, just the ones that were being run sequentially before (`clash` -> `build hdl` -> `simulate hdl`) and where I added `sequenceTest` in this PR. We could add it to `outputTest` too, but I haven't seen it fail ever.

Reply:
You can test it by running the testsuite and see all your cores fire up :cat:

Reply:
Ah, so tests are in parallel, but steps within a test (run clash, run X, run Y, ...) are sequential?

Reply:
Yes. And for the clashLibTests it has the same behavior at the `VHDL`/`Verilog`/`SystemVerilog` level.

Reply:
Sounds good to me